tests/validators/rules/test_rules_validator.py

```diff
@@ -215,6 +215,81 @@ def test_validate_count_operator_non_checkbox_answer():
     assert validator.errors == [expected_error]
 
 
+def test_validate_sum_operator():
+    sum_operator = {"+": [{"source": "answers", "identifier": "array-answer"}]}
+
+    validator = get_validator(
+        sum_operator,
+        answers_with_context={
+            "array-answer": {"answer": {"id": "array-answer", "type": "TextField"}}
+        },
+    )
+    validator.validate()
+
+    expected_error = {
+        "message": validator.ANSWER_TYPE_FOR_SUM_OPERATOR_INVALID.format(
+            answer_type="TextField"
+        ),
+        "referenced_answer": "array-answer",
+        "origin_id": "block-id",
+    }
+
+    assert validator.errors == [expected_error]
+
+
+def test_validate_nested_sum_operator():
+    sum_operator = {
+        "+": [
+            {"+": [{"source": "answers", "identifier": "array-answer"}]},
```

Comment:
Array length can't be one for the numeric operator. Need at least 2 things to add.